config/locales/devise.nl.yml

```diff
@@ -0,0 +1,63 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to en.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+nl:
+  devise:
+    confirmations:
+      confirmed: "Je emailadres is met succes bevestigd."
+      send_instructions: "Je zult een email ontvangen ter bevestiging en verdere instructies binnen enkele minuten."
+      send_paranoid_instructions: "Indien je emailadres reeds in onze database staat, zul je een email ontvangen met instructies om je account te activeren binnen enkele minuten."
+    failure:
+      already_authenticated: "Je bent reeds ingelegd."
+      inactive: "Je account is nog niet geactiveerd."
+      invalid: "Invalide %{authentication_keys} of wachtwoord."
+      locked: "Je account is vergrendeld."
+      last_attempt: "Je hebt nog één poging voordat je account wordt vergrendeld."
+      not_found_in_database: "Invalide %{authentication_keys} of wachtwoord."
+      timeout: "Je sessie is verlopen. Log nogmaals in om door te gaan."
+      unauthenticated: "Je moet inloggen of aanmelden voordat je door kunt gaan."
+      unconfirmed: "Je moet je emailadres bevestigen voordat je door kunt gaan."
+    mailer:
+      confirmation_instructions:
+        subject: "Bevestiging instructies"
```

Comment:
So, while it won't be a direct translation, I think it would be better to say something like "Confirm your account", than "Confirmation instructions".

The translation is technically correct from the English variant, it just sounds really awkward to me. "Confirm your account" in Dutch would be "Bevestig je account" and that would be my suggestion for this.